__tests__/index.test.js

```diff
@@ -338,3 +337,32 @@ describe("PATCH: /api/articles/:article_id tests", () => {
       });
   });
 });
+
+describe("DELETE:/api/comments/:comment_id", () => {
+  test("204: deletes the comment by comment_id", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then((response) => {
+        console.log(response);
+        expect(response.body).toEqual({});
+        expect(response.res.statusMessage).toBe("No Content");
+      });
```

Comment:
This is correct, a 204 has no body to send over. You could even just have this test expecting the 204, but it doesn't hurt to have extra here